### PR TITLE
use a helper for volumes and mounts

### PR DESCRIPTION
**Pull request by joejulian**

### CODE REVIEW

**Comment by alejandroEsc** [comment]
Great reduction of duplication. Could we move this to another helper file, something like _volume_helper.tpl ? the helper file is pretty big as is and it would make those pieces a bit more readable.

File: charts/redpanda/templates/tests/test-kafka-internal-tls-status.yaml [hunk 1]
```diff
@@ -59,26 +59,10 @@ spec:
...
-      volumeMounts:
-        - name: {{ template "redpanda.fullname" . }}
-          mountPath: /tmp/base-config
-        - name: config
-          mountPath: /etc/redpanda
+      volumeMounts: {{ include "default-mounts" . | nindent 8 }}
   {{- range $name, $cert := .Values.tls.certs }}
         - name: redpanda-{{ $name }}-cert
           mountPath: {{ printf "/etc/tls/certs/%s" $name }}
   {{- end }}
```
Comment (RafalKorepta) [inline charts/redpanda/templates/tests/test-kafka-internal-tls-status.yaml:66]: Why this is left behind?

Comment (RafalKorepta) [reply charts/redpanda/templates/tests/test-kafka-internal-tls-status.yaml:66]: I see it being defined in https://github.com/redpanda-data/helm-charts/pull/733/files#diff-8407f32f870c88088a3041f8a5a8394806973dabdef154b7fde83d5e77f70ed8R697-R702

**Review by RafalKorepta — APPROVED** [review]
LGTM

File: charts/redpanda/templates/tests/test-kafka-internal-tls-status.yaml (hunk 1 above)
Comment (joejulian) [reply charts/redpanda/templates/tests/test-kafka-internal-tls-status.yaml:66]: I just really enjoy making CI fail over and over while I go through and find all the mistakes I made. :facepalm:

**Pull request merged**